invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html

```diff
@@ -46,47 +49,63 @@
+
+              </li>
               {%- block navbar_nav %}
                 {%- for item in current_menu.submenu('main').children|sort(attribute='order') if item.visible recursive %}
-                    {%- if item.children %}
-                  <li class="navbar-item navbar-options">
-                    <div class="dropdown{{ ' active' if item.active else '' }}">
-                      <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
-                        aria-expanded="false" href="{{ item.url }}">{{ item.text|safe }} <b class="caret"></b></a>
-                      <ul class="dropdown-menu">{{ loop(item.children|sort(attribute='order')) }}</ul>
-                    </div>
-                  </li>
-                    {%- else %}
-                      <li class={{ "navbar-item navbar-options active" if item.active and loop.depth == 0 else "navbar-item" }}>
+                  {%- if item.children %}
+                    <li class="navbar-item navbar-options">
+                      <div class="dropdown{{ ' active' if item.active else '' }}">
+                        <a class="dropdown-toggle" data-toggle="dropdown" role="button"
+                           aria-haspopup="true"
+                           aria-expanded="false" href="{{ item.url }}">{{ item.text|safe }} <b
+                          class="caret"></b></a>
+                        <ul class="dropdown-menu">{{ loop(item.children|sort(attribute='order')) }}</ul>
+                      </div>
+                    </li>
+                  {%- else %}
+                    <li
+                      class={{ "navbar-item navbar-options active" if item.active and loop.depth == 0 else "navbar-item" }}>
                       <a href="{{ item.url }}">{{ item.text|safe }}</a></li>
-                      {%- endif %}
+                  {%- endif %}
                 {%- endfor %}
-            {%- endblock navbar_nav %}
-            <li class="navbar-item navbar-button">
-              <ul class="rdm-sign-buttons">
-                {%- block navbar_right %}
-                  <li class="navbar-item navbar-button">{%- include config.THEME_HEADER_LOGIN_TEMPLATE %}</li>
-                {%- endblock navbar_right %}
-              </ul>
-            </li>
 
-            <li class="toggle">
-              <input class="menu-btn" type="checkbox" id="menu-btn" />
-              <label class="menu-icon" for="menu-btn"><span class="navicon"></span></label>
-            </li>
-        </ul>
-      </div>
-      </nav>
+              {%- endblock navbar_nav %}
+              <li class="navbar-item navbar-button navbar-actions-container">
+                <ul class="rdm-sign-buttons">
+                  {%- block navbar_right %}
+                    {%- if config.ACCOUNTS and current_user.is_authenticated %}
+                      <li class="navbar-item dashboard-button ">
+                        <a class="navbar-button " href="/dashboard">My dashboard</a>
```

Comment:
I think that this should be injected from the backend and not hardcoded.
We are also not taking into account translations here, which is a must have when adding labels.

Reply:
that makes sense